app/src/main/java/com/example/admin/personallibrarycatalogue/AddBookActivityFragment.java

```diff
@@ -175,7 +197,17 @@ public void onActivityResult(int requestCode, int resultCode, Intent imageReturn
         }
     }
 
-    private void fillAllFields(View view, Book book) {
+    private void fillAllFields(View view, int id) {
+
+        Uri bookWithIdUri = DatabaseContract.BooksTable.buildBookUri(id);
+        Cursor cursor = getActivity().getContentResolver().query(bookWithIdUri,
+                DatabaseContract.BOOK_COLUMNS,
+                bookSelectionQuery_,
+                new String[]{id_.toString()},
+                null);
+
+        Book book = LibraryDatabaseHelper.getBook(cursor);
```

Comment:
Can we create ContentResolverHelper to merge all these three steps into one helper's method called ```getBook```?